### PR TITLE
81 fed update twitter icon in footer

### PR DESCRIPTION
**Pull request by duncansanderson**

### CODE REVIEW

File: src/assets/fontawesome/scss/_animated.scss
Comment (GemmaLouise) [inline src/assets/fontawesome/scss/_animated.scss:1]: Do we need all these SCSS files/fonts? Ideally we only install the FA type we actually need and use for better performance. I think currently we only use one type of icon, if we need more we can add as we go.

File: src/components/footer/components/FooterSocialItem.vue [hunk 1]
```diff
@@ -88,20 +88,36 @@ export default {
...
+
             &.instagram {
                 &:hover {
                     background: $color-instagram;
                     box-shadow: 0 0 0 2px $color-instagram;
                     border-color: $color-instagram;
                 }
             }
+
             &.twitter {
```
Comment (GemmaLouise) [inline src/components/footer/components/FooterSocialItem.vue:100]: Can the old twitter could be removed?

**Review by GemmaLouise — CHANGES_REQUESTED** [review]

File: src/assets/fontawesome/scss/_animated.scss
Comment (duncansanderson) [reply src/assets/fontawesome/scss/_animated.scss:1]: @GemmaLouise good call, I've removed all of the unused FA files

File: src/components/footer/components/FooterSocialItem.vue (hunk 1 above)
Comment (duncansanderson) [reply src/components/footer/components/FooterSocialItem.vue]: I've removed all of the old twitter icon styles

**Review by GemmaLouise — APPROVED** [review]

**Pull request merged**